x-pack/plugins/fleet/public/hooks/use_request/uninstall_commands.ts

```diff
@@ -0,0 +1,31 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License
+ * 2.0; you may not use this file except in compliance with the Elastic License
+ * 2.0.
+ */
+
+import { uninstallTokensRouteService } from '../../../common/services';
```

Comment:
Should this file be renamed to: `get_uninstall_tokens` ?

Reply:
😅 b16d9f8b8eb

although, without `get_` prefix it fits better with the other files in that folder